kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh

```diff
@@ -22,4 +22,7 @@ wlst.sh -skipWLSModuleScanning \
         -managedServerPort ${CUSTOM_MANAGEDSERVER_PORT} \
         -prodMode ${CUSTOM_PRODUCTION_MODE} \
         -managedServerCount ${CUSTOM_MANAGED_SERVER_COUNT} \
-        -clusterName ${CUSTOM_CLUSTER_NAME}
\ No newline at end of file
+        -clusterName ${CUSTOM_CLUSTER_NAME} \
+        -exposeAdminT3Channel ${EXPOSE_T3_CHANNEL_PREFIX} \
+        -t3ChannelPublicAddress ${T3_PUBLIC_ADDRESS} \
```

Comment:
if exposeAdminT3Channel is set to false and the user did not set t3ChannelPublicAddress in the create-domain-input file, the create domain job will fail.
In the log it shows:

```
$ kubectl logs pod/jrfsampledef-create-fmw-infra-sample-domain-job-6cwhs 
The domain will be created using the script /u01/weblogic/create-domain-script.sh

Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

Unexpected argument switch at position 37: 30012
/u01/weblogic/createFMWDomain.py -oh <oracle_home> -jh <java_home> -parent <domain_parent_dir> -name <domain-name> -user <domain-user> -password <domain-password> -rcuDb <rcu-database> -rcuPrefix <rcu-prefix> -rcuSchemaPwd <rcu-schema-password> -adminListenPort <adminListenPort> -adminName <adminName> -managedNameBase <managedNameBase> -managedServerPort <managedServerPort> -prodMode <prodMode> -managedServerCount <managedCount> -clusterName <clusterName>-exposeAdminT3Channel <quoted true or false> -t3ChannelPublicAddress <address of the cluster> -t3ChannelPort <t3 channel port> 

```